sdk/src/azure/core/az_mqtt5_rpc_client_hfsm.c

```diff
@@ -80,8 +80,8 @@ static az_result root(az_event_policy* me, az_event event)
 
     case AZ_HFSM_EVENT_ERROR:
     {
-      _az_RETURN_IF_FAILED(_az_mqtt5_connection_api_callback(
-          this_policy->_internal.connection,
+      _az_RETURN_IF_FAILED(az_event_policy_send_inbound_event(
+          (az_event_policy*)this_policy,
           (az_event){ .type = AZ_HFSM_EVENT_ERROR, .data = event.data }));
       // TODO: uncomment when this won't fail every time
       // if (az_result_failed(az_event_policy_send_inbound_event(me, event)))
```

Comment:
```suggestion
      _az_RETURN_IF_FAILED(az_event_policy_send_inbound_event(
          me,
         event));
```
can remove the rest of that comment as well, github just wouldn't let me select it

Reply:
And for the rest of this file, can remove the commented out send_inbound_event calls and simplify the first parameter of the new send_inbound call